Makefile

```diff
@@ -18,6 +18,7 @@ Eaa_Name := eaa
 Verdict_Name := verdict
 Verdictd_Name := verdictd
 Opa_Name := libopa.so
+RatsTLS_srcdir := $(CUR_DIR)/rats-tls
```

Comment:
Use Rats_Tls_Src_Dir instead, which suits for your variable naming style in this Makefile.